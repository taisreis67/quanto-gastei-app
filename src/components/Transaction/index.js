import  React, { Component } from 'react';

class Transaction extends Component {
    render() {
        return (
            <h1>Transação</h1>
        );
    }
}

export default Transaction;