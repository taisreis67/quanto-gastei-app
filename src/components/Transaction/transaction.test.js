import React from 'react';
import Transaction from '../Transaction';
import { mount } from 'enzyme';
import toJson from 'enzyme-to-json';

it('Deve renderizar os dados da transação informada', () => {
    const props = {
        id: 1,
        valor: 200.00,
        tipo: 1,
        descricao: 'Uma nova transação de débito'
    };

    const transaction = mount(<Transaction {...props} />);

    expect(toJson(transaction)).toMatchSnapshot();
});