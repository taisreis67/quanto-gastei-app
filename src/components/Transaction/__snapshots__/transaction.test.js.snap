// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Deve renderizar os dados da transação informada 1`] = `
<Transaction
  descricao="Uma nova transação de débito"
  id={1}
  tipo={1}
  valor={200}
>
  <h1>
    Transação
  </h1>
</Transaction>
`;
